be/src/common/config.h

```diff
@@ -510,6 +510,8 @@ CONF_mInt32(thrift_rpc_timeout_ms, "5000");
 // txn commit rpc timeout
 CONF_mInt32(txn_commit_rpc_timeout_ms, "20000");
 
+CONF_mInt32(txn_wait_publish_timeout_sec, "1800");
```

Comment:
What's the relation between `txn_wait_publish_timeout_sec` and `wait_apply_time`?

Reply:
> * What's the relation between `txn_wait_publish_timeout_sec` and `wait_apply_time`?
> * What's the behaivor if apply exceed `txn_wait_publish_timeout_sec`?

1. These two configurations are independent. Actually `wait_apply_time` could be deleted if sync publish mechanism are stable. Keep this configure just for safe if sync publish mechanism has some serious issue and we need to disable it.